examples/node_api_indexer/00_get_basic_outputs.rs

```diff
@@ -21,7 +21,7 @@ async fn main() -> Result<()> {
 
     let address = "rms1qpllaj0pyveqfkwxmnngz2c488hfdtmfrj3wfkgxtk4gtyrax0jaxzt70zy";
 
-    // Get output ids of outputs that can be controlled by this address without further unlock constraints
+    // Get output ids of basic outputs that can be controlled by this address without further unlock constraints
```

Comment:
For all the node api examples we do end comments with a "."
So I guess we should do this here too